### ISSUE TITLE
How can I be of help?

### ISSUE THREAD

**Issue by ShoresOfNowhere** [obruns/gelee#4]
To anyone on this team, I'm not such an expert in this kind of programming, but I'm trying to find a solution to the closure of the Gigaset Services, since I was using their Gigaset Elements System.

I'm a skilled web programmer, and I've done a little of industrial automation programming in the past, how can I help the team?

I've already opened up my gigaset elements base (I have two of them, one for the italian market and another for the swiss market), and I also have good soldering tools and skills... 

Please, let me know how can I help! :)

**Comment by ShoresOfNowhere** [comment]
P.s.: If anyone knows how the DECT pairing process can be started on the sensors, I would be grateful knowing it, since I'm currently trying to connect the sensors to a AVM Fritz Smart Gateway, but that doesn't seem to work...

**Comment by jurkov** [comment]
The Gigaset elements dect devices are using a different dect protocol than AVM. it should not work.

**Comment by ShoresOfNowhere** [comment]
@jurkov yes, since posting here I found out they are not a following the standard DECT-ULE protocol...
So the only hope rests in hacking the original Base!
